GitExtensions.ruleset

```diff
@@ -6,7 +6,6 @@
     <Rule Id="CA2101" Action="None" />
   </Rules>
   <Rules AnalyzerId="StyleCop.Analyzers" RuleNamespace="StyleCop.Analyzers">
-    <Rule Id="SA0001" Action="None" />
```

Comment:
Removing the exclusion here effectively enables it.

Reply:
Yes, this is the best way to enable things IMO